analytical_engine/java/grape-runtime/src/main/java/com/alibaba/graphscope/runtime/LoadLibrary.java

```diff
@@ -30,7 +32,11 @@ public class LoadLibrary {
     private static Logger logger = LoggerFactory.getLogger(LoadLibrary.class);
 
     static {
-        System.loadLibrary("grape-jni");
+        try {
+            NativeLoader.loadLibrary("grape-jni");
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
I wonder if the exception got swallowed is by purpose?

Reply:
Yes, we try to load jni library, but it is ok if failed.